package-lock.json

```diff
@@ -1,15 +1,15 @@
 {
```

Comment:
remove this
if you are using pnpm, then use the pnpm-lock.yaml

Reply:
Fixed